lib/mvi/messages/find_candidate_message.rb

```diff
@@ -45,7 +45,7 @@ def validate_types
           errors.add(:given_names, 'should be an array of Strings')
         end
         errors.add(:family_name, 'should be a String') unless @family_name.is_a?(String)
-        errors.add(:dob, 'should be a Time object') unless @dob.is_a?(Time)
+        errors.add(:birth_date, 'should be a Time object') unless @birth_date.is_a?(Time)
```

Comment:
looking at this validation again and I think the user model should ensure it's attributes are valid, going to move it up there (minus type check)